047-TrafficControlWithDapr/Student/Challenge-00.md

```diff
@@ -0,0 +1,323 @@
+    *        aks-dapr-<your value>  aks-dapr-<your value>  clusterUser_rg-dapr-<your value>_aks-dapr-<your value> blah-blah-blah
+    ```
+   
+1.  Install Dapr in your AKS cluster
+
+    Run the following command to initialize Dapr in your Kubernetes cluster using your current context.
+
+    ```shell
+    dapr init -k
+    ```
+
+    Your results should resemble the following:
+
+    ```shell
+    Making the jump to hyperspace...
+    Note: To install Dapr using Helm, see here: https://docs.dapr.io/getting-started/install-dapr-kubernetes/#install-with-helm-advanced
+
+    Deploying the Dapr control plane to your cluster...
+    Success! Dapr has been installed to namespace dapr-system. To verify, run `dapr status -k' in your terminal. To get started, go here: https://aka.ms/dapr-getting-started
+    ```
+
+    Verify the Dapr deployment to your AKS cluster with the following command:
+
+    ```shell
+    dapr status -k
+    ```
+
+    Your results should resemble the following:
+
+    ```shell
+    NAME                   NAMESPACE    HEALTHY  STATUS   REPLICAS  VERSION  AGE  CREATED
+    dapr-sentry            dapr-system  True     Running  1         1.2.2    1m   2021-07-02 08:45.44
+    dapr-sidecar-injector  dapr-system  True     Running  1         1.2.2    1m   2021-07-02 08:45.44
+    dapr-operator          dapr-system  True     Running  1         1.2.2    1m   2021-07-02 08:45.44
+    dapr-dashboard         dapr-system  True     Running  1         0.6.0    1m   2021-07-02 08:45.44
+    dapr-placement-server  dapr-system  True     Running  1         1.2.2    1m   2021-07-02 08:45.45
+    ```
+
+1.  Create the `dapr-trafficcontrol` Kubernetes namespace
+
+    You will need to create a namespace to own all of the TrafficControl Kubernetes objects.
+
+    ```shell
+    kubectl create namespace dapr-trafficcontrol
+    ```
+
+1.  Create Kubernetes secret to allow AKS to pull images from ACR
+
+    You need to grant the AKS cluster access to your Azure Container Registry so that it can pull Docker images. 
+    
+    Get the credentials:
+
+    ```shell
+    az acr credential show -n "<container-registry-name>"
+    ```
+    
+    Run the following command, substituting the username & password you got in the previous step:
+
+    ```shell
+    kubectl create secret docker-registry dapr-acr-pull-secret --namespace dapr-trafficcontrol --docker-server="<container-registry-name>".azurecr.io --docker-username="<container-registry-username>" --docker-password="<container-registry-password>"
+    ```
+
+1.  Assign permissions to KeyVault
+
+    Lastly, assign yourself access to the KeyVault so you can create secrets:
+
+    ```shell
+    az keyvault set-policy --resource-group "<resource-group-name>" --name "<key-vault-name>" --upn "dwight.k.schrute@dunder-mifflin.com" --secret-permissions get list set delete --certificate-permissions get list create delete update
+    ```
+
+### Review sample application architecture
```

Comment:
Is the name of the app "Traffic Application" or 'Traffic Control Application"?   If so, we should probably refer to it as so instead of as "Sample Application".   That applies here and on title of the /Student/Resources/Readme.md file too.